plugin/trino-hive/src/test/java/io/trino/plugin/hive/fs/TestTransactionScopeCachingDirectoryLister.java

```diff
@@ -159,8 +162,8 @@ private static class CountingDirectoryLister
             implements DirectoryLister
     {
         private final Map<Location, List<TrinoFileStatus>> fileStatuses;
-        private int listCount;
-        private boolean throwException;
+        private AtomicInteger listCount;
```

Comment:
Should be `final` and initialized, woops!

Reply:
oops ill fix that :(